src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // LogReader needs to handle older format.
+    // DO this in one central place so that we can remove this class when deprecate the old format. 
+    // One central place 
+    internal class LegacyHack
```

Comment:
nit: I'd rename this to LegacyTableHelper - if this ever were to surface in a stack trace, that name would make people feel better. If we create a bug as I suggested above to remove this, we'll remember w/o the name :)